docs/reference/commandline/network_create.md

```diff
@@ -122,6 +122,26 @@ docker network create -d overlay
 ```
 Be sure that your subnetworks do not overlap. If they do, the network create fails and Engine returns an error.
 
+# Bridge driver options
+
+When creating a custom network, the default network driver (i.e. `bridge`) has additional options that can be passed.
+The following are those options and the equivalent docker flags used for docker0 bridge.
+
+| Option                                           | Equivalent                    | Description                                           |
+|--------------------------------------------------|-------------------------------|-------------------------------------------------------|
+| `com.docker.network.bridge.name`                 | -                             | bridge name to be used when creating the Linux bridge |
+| `com.docker.network.bridge.enable_ip_masquerade` | `docker daemon -ip-masq flag` | Enable IP masquerading                                |
+| `com.docker.network.bridge.enable_icc`           | `docker daemon -icc flag`     | Enable addition of iptables rules                     |
```

Comment:
The Description above should be `Enable or Disable Inter Container Connectivity`.